.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,3 +18,20 @@ If you haven't already, read the full [contribution guide](../CONTRIBUTING.md).
 * [ ] I have opened a new issue for the proposal, or commented on an existing one, and ensured that the Bicep maintainers are good with the design of the feature being implemented
 * [ ] I have included "Fixes #{issue_number}" in the PR description, so GitHub can link to the issue and close it when the PR is merged
 * [ ] I have appropriate test coverage of my new feature
+
+## Contributing a snippet
+
+* [ ] I have only a single resource in my snippet
+* [ ] I have checked that there is not an equivalent snippet already submitted
+* [ ] There should not be syntax errors after inserting a snippet
```

Comment:
Contributors may not be aware of the differences between syntax and semantic errors. Also, is this before or after filling all the placeholders?

Reply:
This was intended for AFTER the snippet is inserted. Would it be better to change it to " There should not be errors after a snippet it inserted"?

Reply:
Yeah I think it's better, but until the placeholders are replaced correctly, you could see errors with any of the snippets. Our snippet test infra also requires replacement text for each placeholder to be specified for each snippet and only asserts no errors after replacements are made.

Reply:
Understood. This is for the _resulting_ snippet, not at author time for the template that includes the tab stops/placeholders.

Your point about the test infra is a good one which makes me think this point is moot.